src/App.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react';
+import { Component, ReactNode } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey : string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  text = 'Nothing was pressed yet';
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.isClicked);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.isClicked);
+  }
+
+  isClicked = (event: KeyboardEvent) => {
```

Comment:
```suggestion
  handleKeyPressed= (event: KeyboardEvent) => {
```
The function should be called with imperative verb and answer the question "What to do?"
Also, press the key, not click